.github/workflows/update_cli_docs.yaml

```diff
@@ -22,9 +23,7 @@ jobs:
         uses: actions/checkout@v3
 
       - name: generate docs
-        run: |
-          go mod download
-          go run .github/gendocs/konstraint.go
+        run: go run -mod=mod .github/gendocs/konstraint.go
```

Comment:
Consider splitting these changes to a separate PR.

Reply:
Made an attempt at this, but then the PR failed without the change -- so to be able to merge with everything passing, we'd need to include it.

I did some rewording of the PR title and the commit messages to paint a better picture.